### DISCUSSION TITLE
How to use 'winget list --command'

### DISCUSSION THREAD

**Discussion by larryclaman** [microsoft/winget-cli#2557] [Q&A]
I see that the 'winget list' command can take a '--command' option, but the documentation is not clear at all about what this is for and how to use it.  The command help simply says: `--command                   Filter results by command`
and at https://learn.microsoft.com/en-us/windows/package-manager/winget/ it similarly says: 
`--command | Filters results by command specified by the application.`

Would appreciate some help/context/examples on what this could be used for.

**Comment by CrayonPosse** [comment]
I'm no expert but from what I can tell it filters based on list of commands or aliases provided by a package. For example, `--command code` matches the `code` command provided by Microsoft Visual Studio Code and `--command aws` matches the `aws` command provided by AWS Command Line Interface.
